pipeline.yml

```diff
@@ -152,13 +154,15 @@ jobs:
         params:
           <<: *vcenter-params
           <<: *os-params
+          <<: *vm-params
+        
 
   - name: clone-base
     plan:
       - get: windows-stemcell-concourse-image
       - get: pipeline-resources
         trigger: true
-        passed: [create-base]
+        passed: [update-base]

```

Comment:
When running the pipeline for the first time, this will be a blocker won't it? `update-base` doesn't run until the after the initial stemcell is created.